Create recording-custom-user-agent (#10551)
DataDog/documentation: content/en/synthetics/guide/recording-custom-user-agent.md

---
title: Record steps with a custom User-Agent
kind: documentation
description: Record Browser test steps with a custom User-Agent string 
further_reading:
- link: "/synthetics/browser_tests/actions"
  tag: "Documentation"
  text: "Learn about steps for browser tests"
- link: "/synthetics/browser_tests/advanced_options/"
  tag: "Documentation"
  text: "Configure advanced options for steps"
---

In some cases, you might need to set a specific `User-Agent` string to be able to record your browser tests steps. To do so:

1. Open your application in a pop up by clickking on **Open in Popup** when on your browser test recorder.
2. Open your Chrome Developer Tools.
3. Click on the menu button with the three vertical dots.
4. Select the **More tools - Network conditions** option.
5. Go to the **Network conditions** tab and disable the **Select automatically** option.
6. Choose **Custom** and enter the `User-Agent` string of interest.

> **Note:** The `User-Agent` string can be overridden at test execution time by adding it as a header in your test configuration. 

Author: ```suggestion
**Note:** The  [default `User-Agent` string][1] can be overridden at test execution time by adding it as a header in your test configuration. 
```

---
title: Record steps with a custom User-Agent
kind: documentation
description: Record Browser test steps with a custom User-Agent string 
further_reading:
- link: "/synthetics/browser_tests/actions"
  tag: "Documentation"
  text: "Learn about steps for browser tests"
- link: "/synthetics/browser_tests/advanced_options/"
  tag: "Documentation"
  text: "Configure advanced options for steps"
---

> ## Overview

Author: ```suggestion
```

> 

Author: ```suggestion
```

> In some cases, you might need to set a specific `User-Agent` string to be able to record your browser tests steps. To do so:

Author: ```suggestion
In some implementations, applications render a certain way only when using a specific `User-Agent` string (e.g., when using a mobile `User-Agent`. In this type of cases, you would need to set the `User-Agent` header to a custom string to be able to record your browser tests steps on your application. To do so:
```

> 1. Open your application in a pop up by clickking on **Open in Popup** when on your browser test recorder.

Reviewer: ```suggestion
1. Open your application in a pop up by clicking on **Open in Popup** in your browser test recorder.
```

1. Open your application in a pop up by clickking on **Open in Popup** when on your browser test recorder.
2. Open your Chrome Developer Tools.
3. Click on the menu button with the three vertical dots.
4. Select the **More tools - Network conditions** option.
5. Go to the **Network conditions** tab and disable the **Select automatically** option.
6. Choose **Custom** and enter the `User-Agent` string of interest.

> **Note:** The User-Agent string can be overriden at test execution time by adding it as a header in your test configuration. 

Author: ```suggestion
**Note:** The `User-Agent` string can be overridden at test execution time by adding it as a header in your test configuration. 
```

> **Note:** The  [default `User-Agent` string][1] can be overridden at test execution time by adding it as a header in your test configuration. 

Reviewer: ```suggestion
**Note:** You can override the [default `User-Agent` string][1] at test execution time by adding it as a header in your test configuration. 
```
